spacy/pipeline/coordinationruler.py

```diff
@@ -0,0 +1,187 @@
+from typing import List, Callable, Optional, Union
+from pydantic import BaseModel, validator
+import re
+
+from ..tokens import Doc
+from ..language import Language
+from ..vocab import Vocab
+from .pipe import Pipe
+
+########### DEFAULT COORDINATION SPLITTING RULES ##############
+ 
+def split_noun_coordination(doc: Doc) -> Union[List[str], None]:
+    """Identifies and splits phrases with multiple nouns, a modifier
+        and a conjunction.
```

Comment:
this doesn't work yet on phrases with multiple modifiers i.e. "juicy apples and rotten oranges" but this is what I'm thinking in terms of how to approach it i.e. a test to ensure the phrase should be split and then a splitting rule

Reply:
FYI @honnibal